examples/layer-browser/src/examples/core-layers.js

```diff
@@ -168,6 +168,7 @@ const ScatterplotLayerExample = {
     getRadius: d => get(d, 'SPACES'),
     opacity: 0.5,
     pickable: true,
+    pickingHighlightColor: [1, 0, 0, 1],
```

Comment:
Colors are 0-255 right? this seems like a very muted color?

I am assuming that the highlight color is the color that should be used to render (highlight) a selected item, and this it is a real color (has nothing to do with picking colors).